ext/odbc/php_odbc.c

```diff
@@ -2378,12 +2418,6 @@ PHP_FUNCTION(odbc_close)
 	link = Z_ODBC_LINK_P(pv_conn);
 	CHECK_ODBC_CONNECTION(link->connection);
 
-	if (link->persistent) {
```

Comment:
this block of code is not needed anymore since `odbc_link_free()` makes sure to remove any trace of the connection